qiskit_nature/transformers/__init__.py

```diff
@@ -16,10 +16,16 @@
 
 .. currentmodule:: qiskit_nature.transformers
 
+Transformers act on a :class:`~qiskit_nature.drivers.QMolecule` to produce an altered copy of it
+as per the specific transformer. So for instance the :class:`FreezeCoreTransformer` will alter the
+integrals and number of particles in a way that freezes the core orbitals out, storing an extracted
```

Comment:
```suggestion
integrals and number of particles in a way that freezes the core orbitals, storing an extracted
```

Also do we want to assume that the reader knows what `core orbitals` are? Otherwise we can explicitly refer to `QMolecule.core_orbitals`.

Reply:
Some new to chemistry may not know - in general in this text its a question of what we need to explain to the set of users that would be using this. Hence in general improving the text to explain things, or at least link to (external) reference material that does, is useful.